tests/RobotFramework/tests/customizing/tedge_init.robot

```diff
@@ -0,0 +1,104 @@
+    Should Match Regexp    ${etc_tedge}    \s*tedge:root\s*
+    ${etc_tedge_mosq}   Execute Command    ls -ld /etc/tedge/mosquitto-conf | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_mosq}    \s*mosquitto:mosquitto\s*
+    ${etc_tedge_operat}   Execute Command    ls -ld /etc/tedge/operations | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_operat}    \s*tedge:tedge\s*    
+    ${etc_tedge_plug}   Execute Command    ls -ld /etc/tedge/plugins | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_plug}    \s*tedge:tedge\s*    
+    ${etc_tedge_devcert}   Execute Command    ls -ld /etc/tedge/device-certs | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_devcert}    \s*mosquitto:mosquitto\s*
+    ${var_tedge}   Execute Command    ls -ld /var/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${var_tedge}    \s*tedge:tedge\s*
+    ${var_log_tedge}   Execute Command    ls -ld /var/log/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${var_log_tedge}    \s*tedge:tedge\s*
+
+Change user/group and check the change
+    [Documentation]    Running tedge init --user <user> --group <group>  is setting custom user/group
+    ...    this test step is confirming the custom values for user/group
+    Execute Command    sudo tedge init --user petertest --group petertest
+    ${etc_tedge}   Execute Command    ls -ld /etc/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge}    \s*petertest:root\s*
+    ${etc_tedge_mosq}   Execute Command    ls -ld /etc/tedge/mosquitto-conf | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_mosq}    \s*mosquitto:mosquitto\s*
+    ${etc_tedge_operat}   Execute Command    ls -ld /etc/tedge/operations | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_operat}    \s*petertest:petertest\s*  
+    ${etc_tedge_plug}   Execute Command    ls -ld /etc/tedge/plugins | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_plug}    \s*petertest:petertest\s*   
+    ${etc_tedge_devcert}   Execute Command    ls -ld /etc/tedge/device-certs | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_devcert}    \s*mosquitto:mosquitto\s*
+    ${var_tedge}   Execute Command    ls -ld /var/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${var_tedge}    \s*petertest:petertest\s*
+    ${var_log_tedge}   Execute Command    ls -ld /var/log/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${var_log_tedge}    \s*petertest:petertest\s*
+
+Tedge init and check if default values are restored
+    [Documentation]    Running tedge init after setting custom user/group should restore the default user/group
+    ...    this test step is confirming the default values for user/group
+    Execute Command    sudo tedge init
+    ${etc_tedge}   Execute Command    ls -ld /etc/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge}    \s*tedge:root\s*
```

Comment:
Can't run the `ls` command and `matching` be moved to a function and just the `directory or file` name to it?